modules/scripts/config.py

```diff
@@ -101,6 +102,7 @@ def _get_env_dict(conf):
     build = conf['build']
     publish = conf['publish']
     extras = conf['extras']
+    social = conf['social']
```

Comment:
Maybe remove the option from the yaml file? Adding `.. timeline:: <account_name>` shouldn't be a lot of work for our users. And it could potentially confuse users who use the manual index as the option in yaml file won't work.